Assets/Scripts/StandardSamples/UI/Peer2Peer/HighFrequency/UIHighFrequencyPeer2PeerMenu.cs

```diff
@@ -89,15 +91,21 @@ protected override void Update()
                 IncomingChat(messageFromPlayer);
             }
 
-            if (currentChatProductUserId == null || !currentChatProductUserId.IsValid())
+            if (!HasValidCurrentProductId())
             {
                 return;
             }
 
-            if(Peer2PeerManager.sendActive && currentChatProductUserId != null)
+            if (Input.GetMouseButtonDown(0))
+            {
+                ParticlesOnClick();
+            }
+
+            if (Peer2PeerManager.sendActive && currentChatProductUserId != null)
```

Comment:
The code above calls `HasValidCurrentProductId`, which already checks this
```suggestion
            if (Peer2PeerManager.sendActive)
```

Reply:
Fixed in the latest commit.